### DISCUSSION TITLE
[UI] "jump to bottom" when at the bottom & also blocking the log

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#11522] [General]
In the new UI (0.14.0 Beta 1) in the log page, there is now a "jump to bottom" button. This is appearing even when you are at the bottom and it is also blocking the log so I can't see the messages.

![2024-05-26-198](https://github.com/blakeblackshear/frigate/assets/8526509/05014bfb-0c17-4755-8f10-47ac0a1562bb)

_My resolution is 2560 x 1440 and the edge browser window is full screen_

**Comment by DrSpaldo** [comment]
PS. I have no idea what that error message means, is there a way for the message to be more user friendly?

**Reply by hawkeye217** [reply]
The error message just means that a websocket connection didn't close cleanly. Nothing to worry about.

**Comment by hawkeye217** [comment]
You can click on the row itself and it will pop open an overlay with more details of the log message.
